tools/aa/commands/ConfigProvider.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 The StartupOS Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.startupos.tools.aa.commands;
+
+import com.google.startupos.common.CommonModule;
+import com.google.startupos.common.FileUtils;
+import com.google.startupos.tools.aa.Protos.Config;
+import dagger.Module;
+import dagger.Provides;
+import javax.inject.Inject;
+
+@Module(includes = CommonModule.class)
+public class ConfigProvider {
+  @Inject
+  public ConfigProvider() {}
+
+  @Provides
+  public static Config getConfig(FileUtils fileUtils, String configFileName) {
+    return (Config)
+        fileUtils.readPrototxtUnchecked(
+            fileUtils.expandHomeDirectory(configFileName), Config.newBuilder());
```

Comment:
Let's add the `expandHomeDirectory()` inside `readPrototxtUnchecked()`

Reply:
@elinabtl I've checked the source, it in fact is already calling it, see impl [here](https://github.com/google/startup-os/blob/master/common/FileUtils.java#L60)